Files Indented Using Tabs Fails to Materialize

Taichi kernels that are indented using tabs (e.g., in `@ti.data_oriented` classes) fails to materialize. This is due to the fact that `remove_indent()` in kernel.py only checks for spaces.

**To Reproduce**
The following snippet uses tabs. Convert the indentation to spaces and it will run fine.
```py
import taichi as ti

@ti.data_oriented
class Test:
	@ti.kernel
	def test(self):
		pass

ti.init(ti.gpu)
test_obj = Test()
test_obj.test()
```

Got this error while compiling on Windows using MSVC:
```
D:\Documents\Projects\taichi\taichi\python\export_lang.cpp : fatal error C1128: number of sections exceeded object file
 format limit: compile with /bigobj [D:\Documents\Projects\taichi\build\taichi_core.vcxproj]
```
The developer installation instructions said that clang should be used, so that may be the issue. However, it was compiling fine before this point, and this does not seem to be a problem that can be fixed by simply switching to clang (unless it uses the 
`/bigobj` option by default)

--- a/python/taichi/lang/kernel_impl.py
+++ b/python/taichi/lang/kernel_impl.py
@@ -25,10 +25,6 @@ if util.has_pytorch():
     import torch
 
 
-def _remove_indent(lines):
-    return textwrap.dedent(lines)
-
-
 def func(fn):
     """Marks a function as callable in Taichi-scope.
 
@@ -155,7 +151,7 @@ class Func:
                 self.taichi_functions[key.instance_id], non_template_args))
 
     def do_compile(self, key, args):
-        src = _remove_indent(oinspect.getsource(self.func))
+        src = textwrap.dedent(oinspect.getsource(self.func))
         tree = ast.parse(src)
 
         func_body = tree.body[0]
@@ -420,7 +416,7 @@ class Kernel:
                                            grad_suffix)
         ti.trace("Compiling kernel {}...".format(kernel_name))
 
-        src = _remove_indent(oinspect.getsource(self.func))
+        src = textwrap.dedent(oinspect.getsource(self.func))
         tree = ast.parse(src)
 
         func_body = tree.body[0]

